src/lib.rs

```diff
@@ -216,18 +216,19 @@ impl Reflection {
 
     pub fn get_compute_group_size(&self) -> Option<(u32, u32, u32)> {
         for inst in self.0.global_inst_iter() {
-            if inst.class.opcode as u32 == 16 {
-                // spirv_headers::Op::ExecutionMode
-                let cs_size = &inst.operands[2..5];
-                use rspirv::dr::Operand::LiteralInt32;
-                if let [LiteralInt32(x), LiteralInt32(y), LiteralInt32(z)] = *cs_size {
+            if inst.class.opcode == spirv::Op::ExecutionMode {
+                use rspirv::dr::Operand::{ExecutionMode, LiteralInt32};
+                if let [ExecutionMode(
+                    spirv::ExecutionMode::LocalSize | spirv::ExecutionMode::LocalSizeHint,
+                ), LiteralInt32(x), LiteralInt32(y), LiteralInt32(z)] = inst.operands[1..]
+                {
                     return Some((x, y, z));
                 } else {
                     // Invalid encoding? Ignoring.
```

Comment:
@hrydgard The precedent of this crate is generally to return `Err` instead of silently ignoring wrong operand combinations (this may indicate a new/changed format down the line, etc).

Perhaps we should make this `fn` return `Result<Option<>>` instead? We can already leverage (albeit slightly misleading) `ReflectError::OperandError` for this.

Reply:
I don't mind either way, I just want the function, thanks for cleaning it up. Didn't expect Jasper to insta-merge without review ;)